File: Polimorfismo/Csharp/Polimorfismo/Animal.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Polimorfismo
{
    public class Animal
    {
        public virtual string MakeSound()
        {
            return "maybe barking dk";
        }
    }

    class Dog : Animal
    {
        public override string MakeSound()
        {
            return "bark bark";
        }
    }

    class Cat : Animal 
    { 
        public override string MakeSound()
        {
            return "meow";
        } 
    }
}

File: Polimorfismo/Python/Animal.py
class Animal:
    def male_sound(self):
        return "some sound"
    
class dog(Animal):
    def make_sound(self):
        return "bark"
    
class cat(Animal):
    def make_sound(self):
        return "mewo mewo"
    
dog = dog()
cat = cat()

print(dog.make_sound())
print(cat.make_sound())